lib/nexpose/site.rb

```diff
@@ -617,6 +617,14 @@ class IPRange
     def initialize(from, to = nil)
```

Comment:
I was concerned about this.  I agree that the Gem and Nexpose UI should handle the same inputs the same way, but ideally they would support CIDR more naturally (which, IMO, means not stripping the first/last).